src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -3,6 +3,142 @@ Ceph Dashboard Developer Documentation
 
 .. contents:: Table of Contents
 
+Preliminary Steps
+-----------------
+
+The following documentation chapters expect a running Ceph Cluster and at
+least a running `dashboard` manager module (with few exceptions). This
+chapter gives an introduction on how to set up such a system for development,
+without the need to set up a full blown production environment. All in this
+chapter introduced options are based on a so called `vstart` environment. 
+
+.. note::
+
+  Every vstart environment needs Ceph `to be compiled`_ from its Github
+  repository, though Docker environments simplify that step by providing a
+  shell script that contains those instructions.
+
+.. _to be compiled: https://docs.ceph.com/docs/master/install/build-ceph/
+
+vstart
+~~~~~~
+
+"vstart" is actually a shell script in the `src/` directory of the Ceph
+repository (``src/vstart.sh``). It is used to start a single node Ceph
+cluster on the machine where it is executed. Several required and some
+optional Ceph internal services are started automatically when it is used to
+start a Ceph cluster. `vstart` is the basis for the three most commonly used
+development environments in Ceph Dashboard.
+
+Host vs Docker
+~~~~~~~~~~~~~~
+
+This document introduces you to three different development environments, all
+based on vstart. Those are:
+
+- vstart running on your host system
+
+- vstart running in a Docker environment
+
+  * ceph-dev-docker_ (mostly used by SUSE employees)
```

Comment:
Suggestion: I would leave out the "mostly used by [...]" part and rather state which OS is supported by the different ceph-dev* environments.